core/shared/src/main/scala/fs2/concurrent/Channel.scala

```diff
@@ -188,6 +194,28 @@ object Channel {
                 (s, rightFalse.pure[F])
           }
 
+        def sendAndClose(a: A) =
```

Comment:
This is a copy-paste of the logic from `send` right? Instead of duplicating it, can we have:

```scala
def send(a: A) = sendImpl(a, false)
def sendAndClose(a: A) = sendImpl(a, true)
def sendImpl(a: A, close: Boolean) = ???
```